geomstats/geometry/_my_manifold.py

```diff
@@ -80,7 +80,7 @@ def belongs(self, point, atol=gs.atol):
         return belongs
 
     # Another example of method of MyManifold.
-    def is_tangent(self, vector, base_point=None):
+    def is_tangent(self, vector, base_point=None, atol=gs.atol):
```

Comment:
Do you want to remove `None` from here too?

Reply:
> Do you think though that the user might not find it weird if they get a missing `base_point` error, even though they know from the math that the `base_point` does not matter to find out if the vector is tangent (for e.g. Euclidean space)? I let you decide there, but maybe for these few manifolds, we could just remove the `base_point` as an argument?

So deepsource does not complaint if the overriding method has `base_point=None` where as the overriden one only has `base_point`, so we can adjust to those where it is mathematically true that base point is not required, which was not the case until now as e.g. hypersphere had bp=None.
And no we shouldn't remove the `base_point` argument otherwise learning algorithm with fail when we passe a base_point!